easybuild/easyconfigs/h/HyPhy/HyPhy-2.5.25-gompi-2022a.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = "CMakeMake"
+
+name = 'HyPhy'
+version = '2.5.25'
```

Comment:
Why such an old version? It's even older than the gompi-2021a variant we already have. 2.5.60 surely?

Reply:
@WilleBell any thoughts on the suggestion from @Micket ?

Reply:
We had a request for this specific version (for some reason I don't really know). But if you think it is too old for this toolchain I can upgrade the version (or downgrade the toolchain).

Reply:
You might want to keep the specific older version for a local repo, and upgrade the version here to the latest one?
That way you meet the request of your user and keep the upstream repo up to date?

Reply:
Makes sense